netqasm/sdk/connection.py

```diff
@@ -616,6 +616,31 @@ def loop_body(
         """
         self._builder.sdk_loop_body(body, stop, start, step, loop_register)
 
+    def while_true(self, max_iterations: int) -> ContextManager[SdkWhileTrueContext]:
+        """Create a context with code to be looped until the exit condition is met, or
+        the maximum number of tries has been reached.
+
+        The code inside the context is automatically looped (re-run).
+        At the end of each iteration, the exit_condition of the context object is
+        checked. If the condition holds, the loop exits. Otherwise the loop
```

Comment:
As you describe it here it's not a while_true but a loop_until.

Reply:
It's indeed a better name. I replaced all `while_true` with `loop_until`.

Reply:
I guess you should also replace WhileTrue in SdkWhileTrueContext then

Reply:
Done